### PR TITLE
test(FTL-17136): integration tests

### PR DESCRIPTION
**Pull request by YoussefAWasfy**

### CODE REVIEW

File: affinidi-messaging-mediator/tests/common.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+pub const MY_DID: &str = "did:peer:2.Vz6MkgWJfVmPELozq6aCycK3CpxHN8Upphn3WSuQkWY6iqsjF.EzQ3shfb7vwQaTJqFkt8nRfo7Nu98tmeYpdDfWgrqQitDaqXRz";
+pub const MEDIATOR_API: &str = "https://localhost:7037/mediator/v1";
+pub const BOB_DID: &str = "did:peer:2.Vz6Mkihn2R3M8nY62EFJ7MAVXu7YxsTnuS5iAhmn3qKJbkdFf.EzQ3shpZRBUtewwzYiueXgDqs1bvGNkSyGoRgsbZJXt3TTb9jD.SeyJ0IjoiZG0iLCJzIjp7InVyaSI6Imh0dHBzOi8vbG9jYWxob3N0OjcwMzcvIiwiYWNjZXB0IjpbImRpZGNvbW0vdjIiXSwicm91dGluZ19rZXlzIjpbXX0sImlkIjpudWxsfQ";
+pub const SECRETS_PATH: &str = "../affinidi-messaging-mediator/conf/secrets.json";
+pub const CONFIG_PATH: &str = "../affinidi-messaging-mediator/conf/mediator.toml";
+
+lazy_static! {
+// Signing and verification key
+pub static ref MY_V1: Value = json!({
```
Comment (anton-iskryzhytskyi) [inline affinidi-messaging-mediator/tests/common.rs:12]: I think the naming shall be improved, as I cannot get where it is used, let's use MEDIATOR (if it is for mediator) or ALICE (if it is for actor that communicates with BOB)

File: affinidi-messaging-mediator/tests/integration_test.rs [hunk 2]
```diff
@@ -0,0 +1,860 @@
...
+        .with_ssl_certificates(&mut vec![
+            "../affinidi-messaging-mediator/conf/keys/client.chain".into(),
+        ])
+        .build()
+        .unwrap();
+
+    let did_resolver = DIDCacheClient::new(ClientConfigBuilder::default().build())
+        .await
+        .unwrap();
+    let my_secrets_resolver = AffinidiSecrets::new(vec![
```
Comment (anton-iskryzhytskyi) [inline affinidi-messaging-mediator/tests/integration_test.rs:74]: WDYT to use alice_secrets_resolver, as it is used as usual to illustrate communication between 2 actors (Alice and Bob)

**Review by anton-iskryzhytskyi — APPROVED** [review]
LGTM!

**Review by stormer78 — APPROVED** [review]
LGTM

**Pull request merged**